### ISSUE TITLE
Programatically generate markdown from code cells | [FR]: add results: asis

### ISSUE THREAD

**Issue by jmbuhr** [quarto-dev/quarto-cli#100]
**tldr**: Regression from Rmd to qmd. `results='asis'` doesn't work in some cases.

I was attempting something similar to what is described in https://quarto.org/docs/computations/running-code.html#jupyter: generating markdown from code cells.
Say we have a folder with pictures and want to generate a revealjs slideshow with those. First, we get all the the image-paths and create markdown that has a heading (to make a new slide) and the image:

```
# Images

```{python}
from pathlib import Path
from IPython.display import display, Markdown

paths = Path('.').rglob('./*.png')
texts = []
for n,p in enumerate(paths):
    text = f'## Image {n}\n\n![This is image {n}]({p})\n\n'
    texts.append(text)
    display(Markdown(text))
```

However, if we look at the intermediate markdown file, we see that the output is enclosed in a div (plus one inner div per display call):
```
::: {.cell}

::: {.cell-output-display}


## Image 0

![This is image 0](./images/image1.png)

:::

... more images

:::

```

Unfortunately, due to the surrounding divs, revealjs no longer recognizes those headings as new slides, though they get formatted as headings and  the images show up, just all on one slide.

With Rmarkdown there was the chunk option: `results = 'asis'` which would include the chunk ouput as plain text in the markdown file, no questions asked. If I do the same in Rmarkdown and add this chunk:

```
```{r test, results='asis'}
for (x in py$texts) {
    cat(x)
}
```

it works, but the same chunk rendered by quarto (with the knitr engine) still encloses the output in a `{.cell}` div.

PS: love what you are doing with quarto!

**Comment by jjallaire** [comment]
Good one! I have revised the behavior to eliminate the excess divs. Updated documentation is here: https://quarto.org/docs/computations/execution-options.html#raw-output

Note that even though the standard syntax is `output: asis`, using `results: asis` will also work for compatibility w/ existing knitr documents.

**Comment by jmbuhr** [comment]
This is amazing, thanks!
